docs/api-compatibility-matrix.md

```diff
@@ -0,0 +1,91 @@
+# API compatibility matrix
+
+## Epics
+
+Milestone 1 (DONE) - <https://scality.atlassian.net/browse/S3C-3768>
+
+Milestone 2 (IN PROGRESS) - <https://scality.atlassian.net/browse/S3C-3938>
+
+Milestone 3 (tickets TBD) - <https://scality.atlassian.net/browse/S3C-4027>
+
+## Object Storage Interoperability Service
+
+Reference API documentation - <https://code.vmware.com/apis/1034>
+
+Legend
+
+* `x` - implemented
+* `.` - not implemented
+* `o` - implemented with default stub
+* `*` - Mandatory APIs to integrate with OSE for essential functions, like basic S3 bucket/object CRUD operations
+
+| API  | Milestone 1 | Milestone 2 | Milestone 3 |
+|:-----|:------------|:------------|-------------|
+| headTenant * | . | . | . |
+| getTenant | . | . | . |
+| updateTenant * | . | . | . |
+| deleteTenant | . | . | . |
+| listTenants * | x | x | x |
+| createTenant * | x | x | x |
+| queryTenants * | x | x | x |
+| queryUsers * | . | x | x |
+| createUser * | . | x | x |
+| listUsers * | . | x | x |
+| getUserWithId * | . | x | x |
+| headUser | . | . | . |
+| updateUserStatus * | . | . | . |
+| deleteUser * | . | . | . |
+| getUserWithCanonicalID * | . | x | x |
+| queryCredentials * | . | . | . |
+| listCredentials * | . | x | x |
+| createCredential * | . | x | x |
+| getCredential * | . | . | . |
+| updateCredentialStatus | . | . | . |
+| deleteCredential | . | . | . |
+| getUsage | o | o | o |
+| getBucketList | . | . | . |
+| getBucketLoggingId | . | . | . |
+| getAnonymousUser | . | . | . |
+| getConsole | . | . | . |
```

Comment:
`getConsole` is partially implemented to always provide only Provider Console URL irrespective of input (https://scality.atlassian.net/browse/S3C-3545) . Returning "Tenant Console URL" i.e., in our case S3 Console URL, based on the input is yet to be implemented (https://scality.atlassian.net/browse/S3C-3546)

Reply:
Changed in [`0b605da` (#47)](https://github.com/scality/vmware-ose-scality/pull/47/commits/0b605da507f27f5507374d44db888ddbf3effdf5)